sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,4 @@
+select "i"."filmId" as "dvd",
+count(*) as "totalDVDS"
+from "inventory" as "i"
+group by "i"."filmId";
```

Comment:
Grouping isn't necessary for this. You just need to count the total number of rows in the "inventory" table.